packages/api-client/src/team/payment/PaymentAPI.ts

```diff
@@ -118,6 +139,9 @@ export class PaymentAPI {
     return response.data;
   }
 
+  /**
+   * @deprecated
```

Comment:
Use BillingAPI? :slightly_smiling_face:

Reply:
There's no replacement for that endpoint because it was never used.